packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -113,7 +119,7 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
 
   T _result;
 
-  void handleStatusChanged(AnimationStatus status) {
+  void _handleStatusChanged(AnimationStatus status) {
```

Comment:
Yeah, I don't see much value in making it public.  Maybe to have a defined ordering w.r.t. other listeners?